features/users/server/methods.coffee

```diff
@@ -222,6 +222,17 @@ Meteor.methods {
                 status: status
         }
 
+    'users.setStatusNoSecurityCheck': (id, status) ->
```

Comment:
This opens quite a big security flaw: any user can update any user status, regardless of their permissions. Why do we want such a thing?